incubator/astro/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.controller.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ include "astro.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "astro.name" . }}
+    helm.sh/chart: {{ include "astro.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: {{ include "astro.fullname" . }}
+subjects:
+  - kind: ServiceAccount
+    name: {{ template "astro.fullname" . }}
```

Comment:
Should this handle the case where serviceAccount.create=false?  It's up to you whether you want to support the combination of `rbac.create=true` and `serviceAccount.create=false`.  Either way, it should be documented.

Reply:
good point - I updated to support a custom serviceaccount name.